PeerioTechnologies/peerio-desktop
# fix: message send error dropdown menu

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/3883/desktop-option-to-retry-sending-message-doesn-t-show

#### Testing instructions  
When user gets message send error (e.g. attempt to send message while offline), clicking the error message should produce the dropdown menu now.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
